docs/en/latest/how-to-build.md

```diff
@@ -125,7 +125,7 @@ Follow the steps below to install Apache APISIX via the source release package.
   cd apisix-${APISIX_VERSION}
   # Create dependencies
   make deps
-  # Install apisix command
+  # Install apisix command (only for luarocks)
   make install
```

Comment:
rm `make install` is better?

Reply:
cc @spacewander

Reply:
Yes. We can rename this task and claim that the developer should not try to "install" the source.

Reply:
@bin-ya I think `Install apisix command (only for luarocks)` should be changed to `Install apisix command (developer do not need to follow this step)`

Reply:
ok